pkgs/tools/text/shab/default.nix

```diff
@@ -0,0 +1,2 @@
+{ pkgs ? import <nixpkgs> {}, ... }:
+pkgs.callPackage ./shab.nix {}
```

Comment:
Why not using `callPackage` directly from the `all-packages.nix` in top-level?

Reply:
thanks for your review, fixed in 2824cd8b6f9b8e200d1187f6369fd3fe6e815ab0

Reply:
I copied most of the nix code from upstream and went too fast :)